site/docs/es/hosting/fly.md

```diff
@@ -0,0 +1,433 @@
+</CodeGroup>
+
+3. **Deno**: Cambiar la versión de Deno y eliminar `CMD` si existe en el archivo `Dockerfile`.
+   Por ejemplo, en este caso, actualizamos `DENO_VERSION` a `1.25.2`.
+
+   **Node.js**: Para cambiar la versión de Node.js, necesitas insertar una propiedad `"node"` dentro de una propiedad `"engines"` dentro de `package.json`.
+   Por ejemplo, actualizamos la versión de Node.js a `16.14.0` en el siguiente ejemplo.
+
+<CodeGroup>
+<CodeGroupItem title="Deno" Active>
+
+```dockerfile{2,26}
+# Dockerfile
+ARG DENO_VERSION=1.25.2
+ARG BIN_IMAGE=denoland/deno:bin-${DENO_VERSION}
+FROM ${BIN_IMAGE} AS bin
+
+FROM frolvlad/alpine-glibc:alpine-3.13
+
+RUN apk --no-cache add ca-certificates
+
+RUN addgroup --gid 1000 deno \
+  && adduser --uid 1000 --disabled-password deno --ingroup deno \
+  && mkdir /deno-dir/ \
+  && chown deno:deno /deno-dir/
+
+ENV DENO_DIR /deno-dir/
+ENV DENO_INSTALL_ROOT /usr/local
+
+ARG DENO_VERSION
+ENV DENO_VERSION=${DENO_VERSION}
+COPY --from=bin /deno /bin/deno
+
+WORKDIR /deno-dir
+COPY . .
+
+ENTRYPOINT ["/bin/deno"]
+# CMD es eliminado
+```
+
+</CodeGroupItem>
+<CodeGroupItem title="Node.js" Active>
+
+```json{19}
+// package.json
+{
+  "name": "grammy",
+  "version": "1.0.0",
+  "description": "grammy",
+  "main": "app.js",
+  "author": "itsmeMario",
+  "license": "MIT",
+  "dependencies": {
+    "express": "^4.18.1",
+    "grammy": "^1.11.0"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.14",
+    "@types/node": "^18.7.18",
+    "typescript": "^4.8.3"
+  },
+  "engines": {
+    "node": "16.14.0"
+  }
+}
+```
+
+</CodeGroupItem>
+</CodeGroup>
+
+4. Edita `app` dentro del archivo `fly.toml`.
+   La ruta `./app.ts` (o `./app.js` para Node.js) en el ejemplo de abajo se refiere al directorio del archivo principal.
+   Puedes modificarlos para que coincidan con el directorio de tu proyecto.
+   Si estás usando webhooks, asegúrate de que el puerto es el mismo que el de tu [configuración](#webhooks) (`8000`).
+
+<CodeGroup>
+<CodeGroupItem title="Deno (Webhooks)" Active>
+
+```toml{7,11,12}
+# fly.toml
+app = "grammy"
+kill_signal = "SIGINT"
+kill_timeout = 5
+
+[processes]
+  app = "run --allow-net ./app.ts"
+
+[[services]]
+  http_checks = []
+  internal_port = 8000
+  processes = ["app"]
+  protocol = "tcp"
+  script_checks = []
+  [services.concurrency]
+    hard_limit = 25
+    soft_limit = 20
+    type = "connections"
+
+  [[services.ports]]
+    force_https = true
+    handlers = ["http"]
+    port = 80
+
+  [[services.ports]]
+    handlers = ["tls", "http"]
+    port = 443
+
+  [[services.tcp_checks]]
+    grace_period = "1s"
+    interval = "15s"
+    restart_limit = 0
+    timeout = "2s"
+```
+
+</CodeGroupItem>
+<CodeGroupItem title="Deno (Long polling)" Active>
+
+```toml{7}
+# fly.toml
+app = "grammy"
+kill_signal = "SIGINT"
+kill_timeout = 5
+
+[processes]
+  app = "run --allow-net ./app.ts"
+
+# Simply omitting the whole [[services]] section 
+# since we are not listening to HTTP
```

Comment:
This should be translated and fit on a single line.

Reply:
Will be fixed in https://github.com/grammyjs/website/pull/558

Thanks